Microsoft.Identity.Web/StartupHelpers.cs

```diff
@@ -1,15 +1,37 @@
+/************************************************************************************************
```

Comment:
Can you use the simplified license header like we do in MSAL? and include the full version as a file? also, the context is different. ex. We don't have a date.